src/rebar_prv_common_test.erl

```diff
@@ -480,52 +517,81 @@ copy_bare_suites(From, To) ->
     ok = rebar_file_utils:cp_r(SrcFiles, To),
     rebar_file_utils:cp_r(DataDirs, To).
 
+maybe_copy_spec(State, [App|Apps], Spec) ->
+    case rebar_file_utils:path_from_ancestor(filename:dirname(Spec), rebar_app_info:dir(App)) of
+        {ok, []}   ->
+            ok = rebar_file_utils:cp_r([Spec],rebar_app_info:out_dir(App));
+        {ok,_} ->
+            ok;
```

Comment:
i don't think we link/copy all directories in an app automatically, so if the spec is in a directory not covered by `src`, `ebin`, `test`, `priv` and `include` it won't be automatically present under `_build`. you may want to translate and create the path and copy manually here